registry/modules/rules_rust/metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "homepage": "https://bazelbuild.github.io/rules_rust/",
   "maintainers": [],
-  "versions": ["0.17.0"],
+  "versions": ["0.22.0"],
```

Comment:
I'm not familiar with bzlmod yet, so it's not clear to me if this registry should have multiple version of dependencies, or just the current one?

Reply:
This registry is only used for testing (see [readme](https://github.com/tweag/rules_nixpkgs/tree/0c1f8f5470c7f292b7620762e224f53d837929d3/registry#readme)), so a single version is enough.